src/node_process.cc

```diff
@@ -59,8 +66,14 @@ void Chdir(const FunctionCallbackInfo<Value>& args) {
   CHECK(args[0]->IsString());
   Utf8Value path(env->isolate(), args[0]);
   int err = uv_chdir(*path);
-  if (err)
-    return env->ThrowUVException(err, "chdir", nullptr, *path, nullptr);
+  if (err) {
+    // Also include the original working directory, since that will usually
+    // be helpful information when debugging a `chdir()` failure.
+    char buf[CHDIR_BUFSIZE];
+    size_t cwd_len = sizeof(buf);
+    uv_cwd(buf, &cwd_len);
```

Comment:
Is it possible that `uv_cwd` here returns something different from the original cwd? (I am thinking about Windows since libuv [calls `SetCurrentDirectoryW`](https://github.com/libuv/libuv/blob/619937c783a05b51fba95cc9a62543deeffe5fa7/src/win/util.c#L291) after `SetCurrentDirectoryW` and `SetEnvironmentVariableW` might fail even after `SetCurrentDirectoryW` succeeds)

Reply:
@joyeecheung I guess so? It’s not really clear to me how `SetEnvironmentVariableW` could fail with a runtime error…

I don’t think we really need to worry about it? It’s going to be pretty rare, I think